packages/components/src/index.tsx

```diff
@@ -44,7 +44,7 @@ interface ICourierConfig {
   unsubscribePage?: {
     topicId: string;
     preferencePageUrl: string;
-    list?: boolean;
+    list?: string;
```

Comment:
how does this fix it? @heyglassy

Reply:
that property is returned as a string from the backend, this will check if it equals the string version of true or false and then pass the boolean to the component.

Reply:
so the issue was occurring for the hosted page because it thought it was always a list by default? or?

Reply:
yes it thought it was list by default because it assumed the backend would pass it a boolean. Because it passed a string, the list was always set to true.